app/presenters/summary/sections/justification_for_legal_aid.rb

```diff
@@ -0,0 +1,36 @@
+module Summary
+  module Sections
+    class JustificationForLegalAid < Sections::BaseSection
+      def name
+        :justification_for_legal_aid
+      end
+
+      def show?
+        kase.present? && super
+      end
+
+      def answers
+        [
+          ioj.types.map do |ioj_type|
+            justification = "#{ioj_type}_justification"
+            Components::FreeTextAnswer.new(
+              :ioj_justification, ioj[justification],
```

Comment:
Here instead of `:ioj_justification` it would just be the type. Or if using the value-object, the `#value` (which is a symbol). Then in the locales we have to declare a key for each of the IoJ types.
Because `ioj_type.humanize` is not really doing localisation.